lib/group_domain_models/lib/src/models/school.dart

```diff
@@ -6,30 +6,31 @@
 //
 // SPDX-License-Identifier: EUPL-1.2
 
-import 'package:meta/meta.dart';
 import 'course.dart';
 import 'member_role.dart';
 
 class School {
-  final String id, name, publicKey, joinLink;
-  final MemberRole myRole;
+  final String? id, name, publicKey, joinLink;
```

Comment:
Can't you make `id` non-nullable? (Are we even using `School`? I don't think so, right?)